src/Entity/Activity.php

```diff
@@ -0,0 +1,222 @@
+<?php
+
+namespace App\Entity;
+
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * @ORM\Entity(repositoryClass="App\Repository\ActivityRepository")
+ */
+class Activity
+{
+    /**
+     * @ORM\Id()
+     * @ORM\GeneratedValue()
+     * @ORM\Column(type="integer")
+     */
+    protected $id;
+
+    /**
+     * @ORM\Column(type="string", length=30)
+     */
+    private $name;
+
+    /**
+     * @ORM\Column(type="string", length=1000)
+     */
+    private $description;
+
+    /**
+     * @ORM\Column(type="datetime")
+     */
+    private $applicationDeadline;
+
+    /**
+     * @ORM\Column(type="datetime")
+     */
+    private $finalDeadline;
+
+    /**
+     * @ORM\Column(type="string", length=30)
+     */
+    private $status;
+
+    /**
+     * @ORM\Column(type="integer")
+     */
+    private $idOwner;
+
+    /**
+     * @ORM\Column(type="datetime")
+     */
+    private $createdAt;
+
+    /**
+     * @ORM\Column(type="datetime")
+     */
+    private $updatedAt;
+
+    /**
+     * @var \Doctrine\Common\Collections\Collection|Technology[]
+     * @ORM\ManyToMany(targetEntity="Technology")
+     */
+    protected $activityTechnologies;
```

Comment:
Why not `$technologies`?

Reply:
@SpartakusMd 
In that case we would have a field named $technologies both in user and activity tables. We used $activityTechnologies and  $userTechnologies instead for a better understanding.